webapp/components/deactivate_member_modal.jsx

```diff
@@ -0,0 +1,113 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import {updateActive} from 'actions/user_actions.jsx';
+import {intlShape, injectIntl, FormattedMessage} from 'react-intl';
+
+import {Modal} from 'react-bootstrap';
+
+import React from 'react';
+
+class DeactivateMemberModal extends React.Component {
```

Comment:
I originally planned on using that component, but the confirmation button has a class of `btn btn-default` rather than `btn btn-danger`. Maybe I could try passing in a class name?